### PR TITLE
Fix updating base classes

### PR DESCRIPTION
**Pull request by DartRuffian**
Fix updating the base classes for various classes.
Currently when launching the game, the following errors will be logged to the rpt:
```sqf
11:40:51 Updating base class Controls->, by x\enh\addons\main\config.bin/Cfg3DEN/Attributes/EditMulti3/Controls/ (original a3\3den\config.bin)
11:40:51 Updating base class Controls->, by x\enh\addons\main\config.bin/Cfg3DEN/Attributes/EditCodeMulti3/Controls/ (original a3\3den\config.bin)
11:40:51 Updating base class EditMulti5->EditCodeMulti3, by x\enh\addons\main\config.bin/Cfg3DEN/Attributes/EditCodeMulti5/ (original a3\3den\config.bin)
11:40:51 Updating base class Controls->Controls, by x\enh\addons\main\config.bin/Cfg3DEN/Attributes/EditCodeMulti5/Controls/ (original a3\3den\config.bin)
11:40:51 Updating base class Background->Background, by x\enh\addons\main\config.bin/Cfg3DEN/Attributes/EditCodeMulti5/Controls/Background/ (original a3\3den\config.bin)
11:40:51 Updating base class Value->Value, by x\enh\addons\main\config.bin/Cfg3DEN/Attributes/EditCodeMulti5/Controls/Value/ (original a3\3den\config.bin)
11:40:51 Updating base class Title->Title, by x\enh\addons\main\config.bin/Cfg3DEN/Attributes/EditCodeMulti5/Controls/Title/ (original a3\3den\config.bin)
11:40:51 Updating base class ValueX->ctrlButton, by x\enh\addons\main\config.bin/Display3DEN/Controls/StatusBar/Controls/Version/ (original a3\3den\config.bin)
11:40:51 Updating base class RscButtonMenu->ButtonSpectatorCamera, by x\enh\addons\main\config.bin/RscDebugConsole/Controls/ButtonFunctions/ (original a3\ui_f\config.bin)
```

This PR fixes these errors by correcting the inheritance. For example, `EditCodeMulti5` is supposed to inherit from `EditMulti5`; however 3den Enhanced incorrectly has it inheriting from `EditCodeMulti3`.

### CODE REVIEW

File: addons/main/cfg3DEN.hpp [hunk 1]
```diff
@@ -107,25 +110,6 @@ class Cfg3DEN
...
-                class Value: Value
-                {
-                    h = QUOTE(20 * 3.5 * GRID_H - 1 * GRID_H);
-                };
-                class Title: Title
-                {
-                    h = QUOTE((20 * 3.5 + 1 * 5) * GRID_H);
-                };
-            };
-        };
```
Comment (DartRuffian) [inline addons/main/cfg3DEN.hpp:128]: See https://github.com/R3voA3/3den-Enhanced/pull/337/commits/55605921bd2cb5876f390c28706c932870d0a130 for details. TL;DR, these same changes are inherited from EditMulti5.

**Comment by R3voA3** [comment]
Thanks for the PR. However, I cannot merge it like that as it breaks removes the height changes of the edit controls and breaks the version button in the status bar.

**Comment by DartRuffian** [comment]
I figured it would break stuff, but I'm not too familiar with all of the things that 3DEN Enhanced touches, so I figured I'd do the inheritance fixes first.

**Comment by R3voA3** [comment]
> I figured it would break stuff, but I'm not too familiar with all of the things that 3DEN Enhanced touches, so I figured I'd do the inheritance fixes first.

All good. I'll look into it later this week.

**Pull request merged**